CHANGELOG.rst

```diff
@@ -4,6 +4,7 @@ Change Log
 3.9 (Unreleased)
 ----------------
 
+- Drop support for Python 2.6, only Python 2.7 is supported
```

Comment:
I'd like to keep the text within ``CHANGELOG.rst`` same as what git-log outputs, the summary line. So, summary line of this patch is ``Drop support for Python 2.6, fixes #111``, it should also appear in ``CHANGELOG.rst`` as well.

I've updated document ``contribution.rst``, let's follow the commit message format documented there. Prefix ``Fix`` could cover most of the cases to mention a fixed issue, but ``Related to`` may also be used if a patch is just relative to an issue rather than fixing that specifically.